oap-server/server-library/library-async-profiler-jfr-parser/pom.xml

```diff
@@ -30,4 +30,25 @@
 
     <artifactId>library-async-profiler-jfr-parser</artifactId>
 
+    <dependencies>
+        <dependency>
+            <groupId>tools.profiler</groupId>
+            <artifactId>async-profiler-converter</artifactId>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
+                <configuration>
+                    <excludes>
+                        **\/JFRConverter.java,**\/Frame.java,**\/Arguments.java,**\/Index.java,**\/Classifier.java,**\/CallStack.java
```

Comment:
I think we should keep the package name included as well. Also, we should keep one class per line in the pom file to improve readability.